content/tutorial-build-run-rippled-ubuntu.md

```diff
@@ -4,7 +4,7 @@
 
 For an overview of `rippled`, see [Operating rippled Servers](tutorial-rippled-setup.html).
 
-Use these instructions to build a `rippled` binary file and run it as a stock `rippled` server on Ubuntu 15.04 or later. These instructions were tested on Ubuntu 16.04 LTS.
+Use these instructions to build a `rippled` 0.90.0+ binary file and run it as a stock `rippled` server on Ubuntu 15.04 or later. ***TODO: Question: I added the specific `rippled` release series to this text - is that okay? Should I remove the reference to Ubuntu 15.04 and just say that this is for Ubuntu 16.04 LTS or later?*** These instructions were tested on Ubuntu 16.04 LTS.
```

Comment:
1. It's good to specify to which `rippled` version(s) these instructions apply. I would change the wording slightly, from:
    > build a `rippled` 0.90.0 binary file and run it as a stock `rippled` server on Ubuntu 15.04 or later.
    
    To:
    
    > build a `rippled` executable from source for version 0.90.0 or higher on supported versions of Ubuntu Linux.
2. Let's just say this is for supported versions of Ubuntu and then specify which verison(s) it was specified in in the next sentence, as you already do in the next sentence.

Reply:
On second thought, maybe let's not use the phrase "supported versions of Ubuntu" because it's not clear whether we mean versions we support specifically for `rippled` or versions Canonical supports in general.

Reply:
I change my recommended rephrase to the following:

> build a `rippled` executable from source for version 0.90.0 or higher on Ubuntu Linux 16.04 or higher.